src/Pages/Home.js

```diff
@@ -4,7 +4,10 @@ import '../Styling/Home.css';
 function Home(){
     return(
         <div className="Home">
-            
+            {/* hero section container  */}
+            <div className="hero_section">
```

Comment:
your div’s className on line 6 starts with a capitol letter and the className on this line starts with a lowercase letter. Make sure to stay consistent with your naming.